spot_wrapper/wrapper.py

```diff
@@ -1275,7 +1275,13 @@ def get_mobility_params(self) -> spot_command_pb2.MobilityParams:
         return self._mobility_params
 
     def velocity_cmd(
-        self, v_x: float, v_y: float, v_rot: float, cmd_duration: float = 0.125, body_height: float = 0.0
+        self,
+        v_x: float,
+        v_y: float,
+        v_rot: float,
+        cmd_duration: float = 0.125,
+        body_height: float = 0.0,
+        use_obstacle_params: typing.Optional[bool] = False,
```

Comment:
nit: This could just be
```suggestion
        use_obstacle_params: bool = False,
```